app/src/main/kotlin/com/getstrm/pace/processing_platforms/ProcessingPlatformViewGenerator.kt

```diff
@@ -208,13 +187,24 @@ abstract class ProcessingPlatformViewGenerator(
             },
             tailOperation = { conditionStep, transform ->
                 val (c, q) = toCase(transform, field)
-                conditionStep.otherwise(q).`as`(field.fullName())
+                conditionStep.otherwise(q).`as`(fieldAlias(field.fullName(), transform))
             },
         )
 
         return caseWhenStatement
     }
 
+    private fun fieldAlias(fullName: String, transform: Transform) = if (transform.hasAggregation()) {
+        field(
+            "{0}_{1}",
+            String::class.java,
+            unquotedName(fullName),
+            unquotedName(transform.aggregation.aggregationTypeCase.name)
```

Comment:
What about casing here? Do you need to lowercase this? No platform will probably fall over this though.

Reply:
does not really matter right? Could do it lowercase, but that is a choice